docs/querying/functions.md

```diff
@@ -185,6 +185,8 @@ for quantiles located in the lowest bucket.
 If `b` contains fewer than two buckets, `NaN` is returned. For φ < 0, `-Inf` is
 returned. For φ > 1, `+Inf` is returned.
 
+If `b` has 0 observations, `NaN` is returned.
```

Comment:
You can put this in with the previous paragraph with the other special case that returns NaN

Reply:
Right. I put it in the same order as the function docs